agent_app_sample_code/utils/file_loading.py

```diff
@@ -1,19 +1,3 @@
-# Databricks notebook source
```

Comment:
Can't import from notebooks - need to make this a normal python file by deleting the "Databricks notebook source" header here